sdk/translation/azure-ai-translation-document/tests/test_translation_async.py

```diff
@@ -523,11 +452,11 @@ async def test_single_input_with_kwargs(self, **kwargs):
         client = kwargs.pop("client")
         variables = kwargs.pop("variables", {})
         # prepare containers and test data
-        source_container_sas_url = self.create_source_container(data=Document(data=b'hello world'), variables=variables)
+        source_container_sas_url = self.create_source_container(data=Document(data=b"hello world"), variables=variables)
         target_container_sas_url = self.create_target_container(variables=variables)
 
         def callback(request):
-            req = _StartTranslationDetails.deserialize(json.loads(request.http_request.body))
+            req = _StartTranslationDetails._deserialize(json.loads(request.http_request.body), [])
```

Comment:
Is this change expected?

Reply:
Yes, because we are now using the _deserialize from _model_base.py.